src/UmbracoFileSystemProviders.Azure/AzureFileSystem.cs

```diff
@@ -540,8 +541,19 @@ public DateTimeOffset GetLastModified(string path)
         /// <returns>
         /// The <see cref="string"/> representing the relative path.
         /// </returns>
+        /// <remarks>
+        /// Umbraco 7.5.15 changed the way that relative paths are used for media upload. 
+        /// This is the fixing issue where uploading file to replace creates new folder.
+        /// </remarks>
         public string GetRelativePath(string fullPathOrUrl)
         {
+            var lastSafeVersion = new Version(7, 5, 14);
```

Comment:
Do we know the exact number? Without we cannot merge this PR as we might end up introduce a bug for older versions.